docs/introduction.rst

```diff
@@ -304,4 +304,60 @@ Cons
 
 Saliency Algorithms
 --------------------------------
-*Discuss the provided xaitk-saliency algorithms in terms of the above.*
+.. Discuss the provided xaitk-saliency algorithms in terms of the above.
+
+The xaitk-saliency API consists of a number of object-oriented interfaces for saliency heatmap generation, which
+focus on black-box visual saliency. Three types of implementations of these interfaces are provided: Image
+Perturbation, Heatmap Generation, and End-to-End Saliency Generation.
+
+
+Image Perturbation
+^^^^^^^^^^^^^^^^^^^
+These implementations are occlusion-based saliency algorithms that perform perturbations on image pixels.
```

Comment:
Technically, there is no saliency at this step (we only generate a set of perturbation masks which is then used for downstream saliency computation):

```suggestion
These implementations are occlusion-based algorithms that perform perturbations on image pixels.
```

Reply:
done